lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -286,10 +286,21 @@ function dosomething_reportback_form_submit($form, &$form_state) {
   dosomething_reportback_save($values);
 
   // Redirect to confirmation.
-  if (!$is_admin_form) {
+  $display_permalink = variable_get('dosomething_display_reportback_permalink');
```

Comment:
Will you add a checkbox to toggle on/off? So we can test staging deploys without this feature. Also.. late feedback but would it be cleaner to prefix the variable with the module that defines it? e.g. `dosomething_reportback_display_permalink`